gulpfile.js

```diff
@@ -11,6 +11,16 @@ gulp.task(
     ),
 );
 
+gulp.task(
+    'ship',
```

Comment:
change this to "deploy" if you don't mind, just so that it matches our other projects

Reply:
done